custom_components/elegoo_printer/image.py

```
@@ -0,0 +1,143 @@
+        super(ElegooPrinterEntity, self).__init__(coordinator=coordinator)
+        self.coordinator = coordinator
+        self.entity_description = description
+        self._attr_content_type = "image/bmp"
+        self._image_filename = "cover_image"
+        self._session = async_get_clientsession(hass)
+        self._attr_unique_id = self.entity_description.key
+
+    async def get_image(self) -> bytes | None:
+        """Update the image from the URL."""
+        url = "http://10.0.0.212:3030/media/mmcblk0p1/history_image/6c4336e6-eecb-11ef-ae89-40f4c926a324.bmp"
+        print(url)
```

Ruff (0.8.2)

`print` found

Remove `print`

(T201)